File: app/src/main/java/com/example/contactapp/MyContacts.java
package com.example.contactapp;

import java.util.ArrayList;


public class MyContacts implements iContact{

    ArrayList<Contact> Contactos = new ArrayList<Contact>();

    @Override
    public void add(Contact contacto) {
        Contactos.add(contacto);
    }

    @Override
    public String phone(Contact contacto) {
        return "Se está llamando a: " + contacto.getNombre();
    }

    @Override
    public String mail(Contact contacto) {
        return "Mi nombre es André Rodríguez y mi número es 56998418";
    }
}

File: app/src/main/java/com/example/contactapp/MyContacts.kt
package com.example.contactapp

import java.util.ArrayList


class MyContacts : iContact {

    internal var Contactos = ArrayList<Contact>()

    override fun add(contacto: Contact) {
        Contactos.add(contacto)
    }

    override fun phone(numero: Int): String {
        return "Se está llamando al: " + Integer.toString(numero)
    }

    override fun mail(destinatario: String): Array<String> {
        return arrayOf(destinatario, "Mi nombre es André Rodríguez y mi número es 56998418")
    }

    override fun getlistas(): ArrayList<String> {
        val result = ArrayList<String>()
        for (contact in Contactos) {
            result.add(contact.nombre + " - " + contact.telefono)
        }
        return result
    }
}
